web/client-api/src/main/java/io/deephaven/web/client/state/ClientTableState.java

```diff
@@ -301,6 +304,8 @@ public void setResolution(ResolutionState resolution, String failMsg) {
             // after a failure, we should discard onRunning (and this state entirely)
             onRunning = null;
             onReleased = null;
+
+            forActiveTables(t -> t.failureHandled(failMsg));
```

Comment:
For one of the two changed sites in `WorkerConnection`, this has (effectively) changed from:
```
                        state.forActiveSubscriptions((table, subscription) -> {
                            table.failureHandled(failMsg);
                        });
```
Is that material?

Reply:
Colin and I discussed. Even though there are some scenarios where the server-side table is still alive after the subscription fails, that the majority of the time it would be best for the client to try to recreate the result table. If the table wasn't alive (at least for our more expensive operations) then memoization will save face, otherwise creating a new table would have been required after a subsequent resubscribe fails.

This is a more conservative approach to the failure scenario, but not exorbitantly so.